lib/clockwork/test/rspec/matchers/have_run.rb

```diff
@@ -0,0 +1,67 @@
+module Clockwork
+  module Test
+    module RSpec
+      module Matchers
+        class HaveRun
+          def initialize(job_name = nil)
+            @job_name = job_name
+            @times_run = 1
+            @exactly = false
+          end
+
+          def matches?(clock_test)
+            if @exactly
+              clock_test.manager.times_run(@job_name) == @times_run
+            else
+              clock_test.manager.ran_job?(@job_name)
+            end
+          end
+
+          def once
+            @times_run = 1
+            @exactly = true
+
+            self
```

Comment:
I suppose the only `self` return I *need* is in `exactly`, so that `times` or `time` could be chained to it.

Written as-is, it would let you do funky things like `have_run(job_name).exactly(2).times.once`...which would really just check if it was called once. I suppose we don't really want to encourage or allow that behavior.